packages/instant/src/types.ts

```diff
@@ -95,7 +95,7 @@ export enum Network {
 
 export enum ZeroExInstantError {
     AssetMetaDataNotAvailable = 'ASSET_META_DATA_NOT_AVAILABLE',
-    InsufficientETH = 'INSUFFICIENT_ETH',
+    InsufficientEth = 'INSUFFICIENT_ETH',
```

Comment:
Can we use `// tslint:disable-line:enum-naming` here, as we use it above to keep ETH as `ETH` instead of `Eth` ? 

Related: I like the @LogvinovLeon 's idea of having a config for known/whitelisted acronyms, but understand that it could be a follow-up